src/components/Drawer/Drawer.jsx

```diff
@@ -22,12 +22,30 @@ function Drawer() {
       >
         <DrawerIcons name="History" type={'HISTORY'} size={'25px'} />
       </NavLink>
-      <NavLink to="/liked">
+      <NavLink
+        style={({ isActive }) => {
+          return isActive ? { color: 'var(--primary-color)' } : null;
+        }}
```

Comment:
since this function is getting repeated, you can extract it out

Reply:
True. will make sure to implement these changes.